.github/workflows/react.yml

```
@@ -377,7 +381,7 @@ jobs:
       - storybook_eslint
       - storybook_stylelint
       - storybook_compile
-    if: ${{ needs.setup.outputs.deploy == 'true' }}
+    if: ${{ needs.setup.outputs.deploy == 'true' || needs.setup.outputs.is_pr == 'true' }}
```

Je pense que du dois faire de quoi du genre vu que le `react_deploy` est skipped :
```suggestion
    if: |
      always() &&
      !failure() &&
      !cancelled() &&
      ${{ needs.setup.outputs.deploy == 'true' || needs.setup.outputs.is_pr == 'true' }} &&
      !contains(needs.*.result, 'failure') &&
      !contains(needs.*.result, 'cancelled')
```

Je suis pas certain de la syntaxe pour `${{ needs.setup.outputs.deploy == 'true' || needs.setup.outputs.is_pr == 'true' }} &&`. Je sais pas si tu peux changer pour `(needs.setup.outputs.deploy == 'true' || needs.setup.outputs.is_pr == 'true') &&`

---

J'avais pas remarqué la dépendance sur `react_deploy`.
On pourrait simplement la retirer?

---

Je pense qu'on a la dépendance pour pas déployer le storybook sur les releases si le deploy de la version n'a pas fonctionné

---

Bon point. Je vais essayer ta suggestion.

---

Pour ton questionnement par rapport à la syntaxe, normalement le `${{ }}` devrait encapsuler toute l'expression, mais il est optionnel dans un `if`.